source/renderer/app/containers/wallet/WalletReceivePage.js

```diff
@@ -84,29 +85,61 @@ export default class WalletReceivePage extends Component<Props, State> {
     });
   };
 
+  handleGenerateAddress = (spendingPassword: ?string) => {
+    const { activeWallet } = this.state;
```

Comment:
Do we need to use state, here?

Reply:
@daniloprates we don't need but because we are using `activeWallet` inside render and outside in functions, to prevent multiple declarations from the store, `activeWallet` is set once as constant in the state as `activeWallet: this.props.stores.wallets.active`. If you think that we still need to change that I will make an improvement?

Reply:
It's not a big problem, if we're a rush we can leave it, but it'd better not to use state when we can simply use props.